### DISCUSSION TITLE
Bltouch not deploying SOLVED

### DISCUSSION THREAD

**Discussion by QuinnWede** [Jyers/Marlin#1054] [Q&A] [answered]
Hi all, I've got a bltouch v3.1 on my ender 3 v2 and it self tests just fine when the printer is powered on but when homing, the BLTouch never deploys with a solid red LED on and I have to stop it as it will crash into the bed. The BLTouch is wired into the dedicated port. I am using Jyers firmware "E3V2-BLTouch-3x3-v4.2.7-v1.3.4"

Thanks in advance and let me know if you need any other info to figure this out.

Edit: The board's pinout was labeled completely backward's

![303qgz3wry071](https://user-images.githubusercontent.com/74682662/123160077-9f359e80-d422-11eb-8883-008ce9c5f50c.jpg)
![v0wc9mypry071](https://user-images.githubusercontent.com/74682662/123160081-a066cb80-d422-11eb-97ab-87d333ffc013.jpg)

**Comment by agentmcmillan** [comment, marked as answer]
you need the other cable to make this work.

**Reply by smooreace** [reply]
That cable will work just as well.. its just different plastic on one end. Just have to ensure that the wires are in the right order.
